k9mail/src/main/res/values/arrays.xml

```diff
@@ -512,6 +512,18 @@
         <item>0</item>
     </string-array>
 
+    <string-array name="account_settings_attachment_resize_factor_entries">
+        <item >@string/account_settings_attachments_resize_factor_entry_original</item>
+        <item >@string/account_settings_attachments_resize_factor_entry_half</item>
+        <item >@string/account_settings_attachments_resize_factor_entry_one_fourth</item>
+    </string-array>
+
+    <string-array name="account_settings_attachment_resize_factor_values">
+        <item>1</item>
```

Comment:
The numbers here are fairly arbitrary. They refer to constants in the code. That's not very obvious.

Instead you can:

1. Declare an enum in Account:

```
ResizeFactor {
    STANDARD, HALF, QUARTER
}
```

2. Use the enum constant's names in account_settings_attachment_resize_factor_values instead of 1, 2 and 4:

2. Use the enum constants everywhere else to switch on.

You can see an example of this in `setup/AccountSettings.java`'s `mFolderMode`.